apps/emqx/test/emqx_channel_hookpoints_SUITE.erl

```diff
@@ -48,10 +48,44 @@ t_delayed_puback(_Config) ->
     ),
     emqtt:disconnect(ConnPid).
 
+t_deliver_via_handle_info({init, Config}) ->
```

Comment:
Please add a comment describing what property is being verified and how; what is given and what is expected.